cloudflare-ddns.sh

```diff
@@ -1,10 +1,17 @@
 #/usr/bin/env sh
 
-AUTH_EMAIL=example@example.com
-AUTH_KEY=** CF Authorization  key **
-ZONE_ID=** CF Zone ID **
-A_RECORD_NAME="dynamic"
-A_RECORD_ID=** CF A-record ID from cloudflare-dns-id.sh **
+source ./config
+
+
+A_RECORD_ID=($(curl -X GET "https://api.cloudflare.com/client/v4/zones/$CLOUDFLARE_ZONE_ID/dns_records?type=A&name=$CLOUDFLARE_RECORD_NAME" \
+     -H "Host: api.cloudflare.com" \
+     -H "User-Agent: ddclient/3.9.0" \
+     -H "Connection: close" \
+     -H "X-Auth-Email: ${CLOUDFLARE_EMAIL}" \
+     -H "X-Auth-Key: ${CLOUDFLARE_KEY}" \
+     -H "Content-Type: application/json" -s | jq -r ".result[].id"))
```

Comment:
This would create an extra API call every time the cronjob runs... I think it should be pre fetched like it is now

Btw, I did almost the same thing and created other PR here
https://github.com/dcerisano/cloudflare-dynamic-dns/pull/6

Reply:
What's the issue with an extra API call? Sounds like you're trying to over-optimise for a non-issue here. Sure, saving 300ms on the request and a bit of electricity is nice, but...

Reply:
Instead of merging the tow, it would probably be better to create a separate config file that updates and calls the id script, then updates and kicks off the cron job. Separation of concerns.